osu.Game/Skinning/LegacySkin.cs

```diff
@@ -81,15 +89,31 @@ public override Drawable GetDrawableComponent(string componentName)
                         };
             }
 
-            // temporary allowance is given for skins the fact that stable handles non-animatable items such as hitcircles (incorrectly)
-            // by (incorrectly) displaying the first frame of animation rather than the non-animated version.
-            // users have used this to "hide" certain elements like hit300.
-            var texture = GetTexture($"{componentName}-0") ?? GetTexture(componentName);
+            var texture = GetTexture($"{componentName}-0");
 
-            if (texture == null)
-                return null;
+            if (texture != null && animatable)
+            {
+                var animation = new TextureAnimation { DefaultFrameLength = animationData.frametime };
+
+                for (int i = 1; texture != null; i++)
+                {
+                    animation.AddFrame(texture);
+                    texture = GetTexture($"{componentName}-{i}");
+                }
+
+                animation.Repeat = animationData.looping;
```

Comment:
is there a looping animation in skins? if not, this line can be removed

Reply:
i believe looping is default behaviour

Reply:
Yes, there are definitely looping animations in stable (e.g. animated back buttons)

Reply:
Also slider ball and basically every skinnable element, so I agree it should probably be on by default.